backend/api/Models/Concepts/Document/DocumentTypeMap.cs

```diff
@@ -0,0 +1,19 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts.Document
+{
+    public class DocumentTypeMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsDocumentTyp, DocumentTypeModel>()
+                .PreserveReference(true)
+                .Map(dest => dest.Id, src => src.DocumentTypeId)
+                .Map(dest => dest.DocumentType, src => src.DocumentType)
+                .Map(dest => dest.RowVersion, src => src.ConcurrencyControlNumber)
+                .Inherits<Entity.IBaseAppEntity, BaseAppModel>();
```

Comment:
I think the RowVersion mapping is done by the  .Inherits<Entity.IBaseAppEntity, BaseAppModel>(); line

Reply:
Done